packages/flutter_tools/lib/src/commands/create_base.dart

```diff
@@ -375,6 +376,8 @@ abstract class CreateBase extends FlutterCommand {
       'withPluginHook': withPluginHook,
       'androidLanguage': androidLanguage,
       'iosLanguage': iosLanguage,
+      'hasIosDevelopmentTeam': iosDevelopmentTeam != null && iosDevelopmentTeam.isNotEmpty,
```

Comment:
Can just be `'hasIosDevelopmentTeam': iosDevelopmentTeam.isNotEmpty` if taking my suggestion above.